### PR TITLE
Test out sub-module calling

### PR DESCRIPTION
**Pull request by grw300**
TL;DR: Greg learned something! You should use a "main" or "driver" script for most projects - e.g. `main.py`

Explain:
When you call `python <something.py>` the *actual* name of the `something` module at runtime, no matter what, is "main" regardless of the file name or folder name. See [these docs](https://docs.python.org/3.6/tutorial/modules.html#executing-modules-as-scripts). This means that if you run a sub-module (e.g. `db.cur`) as a *script* it won't see the folder structure above it; it's the "main" module, not `est` - that module is unknown.

Example run of these test modules:
IMG_0387

### CODE REVIEW

**Comment by owenmwilliams** [comment]
Got it, so if I run the script from module level it'll work because that defines it's working tree? Even though sub-modules call each other?

**Comment by grw300** [comment]
That's my new understanding. But clearly my knowledge of the module structure in python is not complete, so there may be more wrinkles to this story!
